syft/codes.py

```diff
@@ -0,0 +1,7 @@
+class MSGTYPE(object):
```

Comment:
Can you add a comment describing the file, and a brief comment defining the class?